docs/ref/contrib/admin/index.txt

```diff
@@ -2904,6 +2910,44 @@ The presence of the ``admin_password_reset`` named URL will cause a "forgotten
 your password?" link to appear on the default admin log-in page under the
 password box.
 
+.. _overriding-default-adminsite:
+
+Overriding the default ``AdminSite``
+------------------------------------
+
+.. versionadded:: 2.1
+
+You can override the default `django.contrib.admin.sites.AdminSite` instance with your own:
```

Comment:
maybe `instance` should be `class` here. (you can't use a custom instance, only a custom class)

Reply:
What about reword it with:
```
You can override the default `django.contrib.admin.site` with your own:
set the `default_site` attribute to the dotted import path with your custom
`AdminSite` subclass or with a callable which returns your site instance.
```
What do you think?

Reply:
Looks good to me. Thanks!